app/views/layouts/application.html.haml

```diff
@@ -28,8 +28,9 @@
             .brandbar.default
               .b1
               .b2
-          = #render partial: 'feedback_and_menu'
-
+          .feedback{tabindex:1}
+            %a.button--feedback{href:ENV.fetch('FEEDBACK_URL','')+request.url, role:"button"}
```

Comment:
There's no requirement given, but I wonder if FEEDBACK_URL is empty should we just not show it at all?